docs/hosting/server-setups/digital-ocean.md

```diff
@@ -1,24 +1,37 @@
+This hosting guide shows you how to self-host n8n on a DigitalOcean droplet. It uses:
+
+* [Caddy](http://caddyserver.com){:target="_blank" .external-link} (a reverse proxy) to allow access to the Droplet from the internet. 
+* [Docker Compose](https://docs.docker.com/compose/){:target="_blank" .external-link} to create and define the application components and how they work together.
+* The [n8n Docker image](https://hub.docker.com/r/n8nio/n8n){:target=_blank .external-link}.
 
 ## Create a Droplet
 
-Log in to DigitalOcean and select the project to host the Droplet and click **Droplets** from the **Manage** menu. You can use [the Docker image](https://marketplace.digitalocean.com/apps/docker){:target="_blank" class=.external-link} from the **Marketplace** tab or use one of the Linux distributions and [install Docker yourself](https://www.docker.com/get-started/){:target="_blank" class=.external-link}. The rest of this tutorial assumes you have Docker installed on the Droplet.
+1. [Log in](https://cloud.digitalocean.com/login){:target=_blank .external-link} to DigitalOcean. 
+2. Select the project to host the Droplet, or [create a new project](https://docs.digitalocean.com/products/projects/how-to/create/){:target=_blank .external-link}.
+3. In your project, select **Droplets** from the **Manage** menu. 
+4. [Create a new Droplet](https://docs.digitalocean.com/products/droplets/how-to/create/){:target=_blank .external-link} using the [Docker image](https://marketplace.digitalocean.com/apps/docker){:target="_blank" .external-link} available on the **Marketplace** tab.
+
+!!! note "Droplet resources"
+		When creating the Droplet, DigitalOcean asks you to choose a plan. For most usage levels, a basic shared CPU plan is enough.
+
+!!! note "SSH or Password"
+		DigitalOcean allows you to choose between SSH and password-based authentication. SSH is more secure. The rest of this guide assumes you are using SSH.
 
 ## Log in to your Droplet
 
-The remainder of the steps in this guide require you to log in to the Droplet using a terminal with SSH. [Find more details on how to do this in the Digital Ocean documentation](https://docs.digitalocean.com/products/droplets/how-to/connect-with-ssh/){:target="_blank"}.
+The rest of this guide requires you to log in to the Droplet using a terminal with SSH. Refer to [How to Connect to Droplets with SSH](https://docs.digitalocean.com/products/droplets/how-to/connect-with-ssh/){:target="_blank" .external-link} for guidance.
 
 ## Create folders and files
 
 Both n8n and Caddy require creating folders that the host operating system (the DigitalOcean Droplet) copies to Docker containers to make them available to Docker.
 
-Create the following on the Droplet in your home folder:
+Create the following in the Droplet in your home folder:
```

Comment:
Not sure this is correct… I always thought it was "on", but I also can't seem to find anything definitive… It's a server, and typically you say "on a server", but I think I was more trying to avoid repeated "on"s.

Maybe…

```suggestion
Create the following in the Droplet root user's home folder:
```

Which is actually also more accurate now that I think about it.